clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -20,6 +26,7 @@ import org.broadinstitute.clio.util.model.UpsertId
   */
 class ElasticsearchIndex[CI <: ClioIndex](
   val clioIndex: CI,
+  val defaults: Json,
```

Comment:
@danxmoran Is this more what you were envisioning?

Reply:
Ye!

If we want to constrain it even more, we could make `defaults` be `clioIndex.MetadataType` and then do the `asJson` in the class body. But I think there's some arbitrary restriction on constructor parameters referring to one another that might block it...

Reply:
I think there is. I'm not too happy about defining the actual default values in `ElasticSearchIndex` either. I don't think there should be data in our mapping logic.